### PR TITLE
x11-libs/wxGTK: fix ld.lld errors: version script assignment failed: symbol not defined

### PR DESCRIPTION
**Pull request by AngryLoki**
ld.lld 17 fix, similar to https://github.com/gentoo/gentoo/pull/34595

Closes: https://bugs.gentoo.org/915154

### CODE REVIEW

File: x11-libs/wxGTK/wxGTK-3.0.4-r5.ebuild [hunk 1]
```diff
@@ -53,6 +53,13 @@ PATCHES=(
 )
 
 multilib_src_configure() {
+	# ideally we want !tc-ld-is-bfd for best future-proofing, but it needs
+	# https://github.com/gentoo/gentoo/pull/28355
+	# mold needs this too but right now tc-ld-is-mold is also not available
+	if tc-ld-is-lld; then
```
Comment (thesamesam) [inline x11-libs/wxGTK/wxGTK-3.0.4-r5.ebuild:59]: Please use the new way of writing this: `append-ldflags $(test-flags-CCLD -Wl,--undefined-version)`.

Existing packages have already been updated.

Comment (thesamesam) [reply x11-libs/wxGTK/wxGTK-3.0.4-r5.ebuild:59]: Also, add a "bug #123456" ref in the comment.

Comment (AngryLoki) [reply x11-libs/wxGTK/wxGTK-3.0.4-r5.ebuild]: Thank you for review! Please note that there are a bunch of packages with old block: https://github.com/search?q=repo%3Agentoo%2Fgentoo%20%22ideally%20we%20want%22&type=code

Comment (thesamesam) [reply x11-libs/wxGTK/wxGTK-3.0.4-r5.ebuild]: I fixed most of those in ~arch but not in stable fwiw

**Pull request closed without merging**